cerbos/sdk/model.py

```diff
@@ -218,6 +218,7 @@ class PlanResourcesResponse:
     resource_kind: str
     policy_version: str
     filter: Optional[PlanResourcesFilter] = None
+    validation_errors: Optional[List[ValidationError]] = None
```

Comment:
should the list be optional here? is there a meaningful difference between none and some(empty list)?